src/components/AddContainerDialogComponent.vue

```diff
@@ -1,28 +1,42 @@
 <template>
-  <Dialog v-model:visible="visible" :header="$t('c_POSCreate.add container')">
+  <Dialog v-model:visible="visible"
+          :header="$t('c_POSCreate.add container')"
+          :draggable="false"
+          modal
+          @update:visible="closeDialog"
+  >
     <div class="dialog">
-      <div class="row">
-        <h6>{{ $t("c_containerEditModal.Name") }}</h6>
-        <InputText class="flex-child" v-model="name" />
-      </div>
-      <div class="row">
-        <h6>{{ $t("c_containerEditModal.owner") }}</h6>
-        <Dropdown
-            class="flex-child"
-            :options="organsList"
-            optionLabel="firstName"
-            v-model="selectedOwner"
-            :placeholder="$t('c_containerEditModal.select owner')"
-        />
-      </div>
-      <div class="row">
-        <h6>{{ $t("c_containerEditModal.Public") }}</h6>
-        <Checkbox class="flex-child" :binary="true" v-model="isPublic"/>
-      </div>
-      <div class="row" id="actions">
-        <Button severity="danger" outlined @click="visible = false">{{ $t("c_containerEditModal.cancel") }}</Button>
-        <Button severity="danger" @click="saveContainer">{{ $t("c_containerEditModal.save") }}</Button>
-      </div>
+      <form @submit="handleCreateContainer">
```

Comment:
Why the gigantic distance between name and the input field? also they dont seem alligned but it is difficult to see.

Reply:
The gigantic distance is fixed now. They are fully aligned :)